flocker/acceptance/endtoend/test_installer.py

```diff
@@ -0,0 +1,319 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for AWS CloudFormation installer.
+"""
+
+from datetime import timedelta
+import json
+import os
+from subprocess import check_call, check_output
+import time
+
+from twisted.internet import reactor
+from twisted.internet.task import deferLater
+from twisted.internet.defer import maybeDeferred
+
+from twisted.python.filepath import FilePath
+from eliot import Message
+
+
+from ...common.runner import run_ssh
+from ...common import gather_deferreds
+from ...testtools import AsyncTestCase, async_runner
+from ..testtools import Cluster, ControlService
+from ...ca import treq_with_authentication, UserCredential
+from ...apiclient import FlockerClient
+from ...control.httpapi import REST_API_PORT
+
+REGION = os.environ['REGION']
+ACCESS_KEY_ID = os.environ['ACCESS_KEY_ID']
+SECRET_ACCESS_KEY = os.environ['SECRET_ACCESS_KEY']
+
+PARAMETERS = [
+    {
+        'ParameterKey': 'KeyName',
+        'ParameterValue': os.environ['KEY_NAME']
+    },
+    {
+        'ParameterKey': 'AccessKeyID',
+        'ParameterValue': os.environ['ACCESS_KEY_ID']
+    },
+    {
+        'ParameterKey': 'SecretAccessKey',
+        'ParameterValue': os.environ['SECRET_ACCESS_KEY']
+    }
+]
+
+COMPOSE_NODE0 = '/home/ubuntu/postgres/docker-compose-node0.yml'
+COMPOSE_NODE1 = '/home/ubuntu/postgres/docker-compose-node1.yml'
```

Comment:
:question: Maybe worth adding a comment that this installer only works on Ubuntu for now.